mltrace/entities/history.py

```diff
@@ -0,0 +1,73 @@
+
+
+import copy
+from datetime import datetime
+
+from mltrace.db import Store
+from mltrace import utils as clientUtils
+from mltrace.entities import base
+
+
+class History(object):
+    def __init__(
+        self,
+        componentName,
+    ):
+        self.component_name = componentName
+
+    def get_runs_by_time(
+        self,
+        start_time: datetime,
+        end_time: datetime,
+    ):
+        store = Store(clientUtils.get_db_uri())
```

Comment:
Instead of initializing a new `store` object in every method. you can initialize a `self.store` in the constructor for `History`